base/replutil.jl

```diff
@@ -248,7 +248,13 @@ showerror(io::IO, ::DivideError) = print(io, "DivideError: integer division erro
 showerror(io::IO, ::StackOverflowError) = print(io, "StackOverflowError:")
 showerror(io::IO, ::UndefRefError) = print(io, "UndefRefError: access to undefined reference")
 showerror(io::IO, ::EOFError) = print(io, "EOFError: read end of file")
-showerror(io::IO, ex::ErrorException) = print(io, ex.msg)
+function showerror(io::IO, ex::ErrorException)
+    print(io, ex.msg)
+    if ex.msg == "type String has no field data"
```

Comment:
It will have a backtrace.

I don't think the performance of showerror is likely to matter.